applications/dashboard/views/settings/registration.php

```diff
@@ -55,6 +55,7 @@
                 </tbody>
             </table>
         </li>
+        <?php if (c('Garden.Registration.ManageCaptcha', true)) : ?>
```

Comment:
Since you're checking `c('Garden.Registration.ManageCaptcha', true)` in the controller, its value should be assigned to the data array and checked there, instead of a) double checking the config in the view and b) making the view depend on the config